source/_data/plugins.yml

```diff
@@ -2496,3 +2496,10 @@
     - server
     - editor
     - preview
+- name: hexo-cloudinary
+  description: Hexo tag to add an image from Cloudinary using Cloudinary SDK.
+  link: https://github.com/maliMirkec/hexo-cloudinary
```

Comment:
Hum... both entries have the same source `https://github.com/maliMirkec/hexo-cloudinary` too

Reply:
put `https://github.com/maliMirkec/hexo-cldnry` instead

Reply:
Done. Thank you.